crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -285,8 +285,9 @@ def restore_database
       end
 
       def proposal?(file)
-        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
-          file.to_s =~ /knife\/databags\/crowbar/
+        !file.basename.to_s.match(
+          /(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)|(^swift\.json$)/
+        ) && file.to_s =~ /knife\/databags\/crowbar/
```

Comment:
In the dump they still have the `bc-` prefix, that's correct. but in `upgrade.rb` which runs before restore, they are renamed. https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/lib/crowbar/upgrade.rb#L96